lib/Doctrine/ORM/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1980,11 +1968,29 @@ private function getValues($value)
      */
     private function getIndividualValue($value)
     {
-        if ( ! is_object($value) || ! $this->em->getMetadataFactory()->hasMetadataFor(ClassUtils::getClass($value))) {
-            return $value;
+        if ( ! is_object($value)) {
```

Comment:
I think git-phpcs is indicating the wrong line to cs2pr. This should fix the cs issue.

```suggestion
        if (! is_object($value)) {
```